com.unity.render-pipelines.universal/Shaders/BakedLitDepthNormalsPass.hlsl

```diff
@@ -59,6 +58,8 @@ float4 DepthNormalsFragment(Varyings input) : SV_TARGET
     half alpha = texColor.a * _BaseColor.a;
     AlphaDiscard(alpha, _Cutoff);
 
+    LODFadeCrossFade(input.positionCS);
```

Comment:
For some reason I have to put this always after alpha discard otherwise there is a weird behavior on iOS.